mercury-operator/src/test/resources/application.properties

```diff
@@ -1 +1,2 @@
-quarkus.operator-sdk.crd.apply=true
\ No newline at end of file
+quarkus.operator-sdk.crd.apply=true
```

Comment:
I think the crds should only be applied in dev/test mode as it should be part of the installation

Reply:
it is part of the application.properties in the test resources

Reply:
oh true, I thought it was the other one